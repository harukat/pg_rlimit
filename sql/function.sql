SELECT * FROM pg_setrlimit('v', -1);
SELECT * FROM pg_getrlimit('v');
SELECT * FROM pg_setrlimit('v', 512 * 1024 * 1024);
SELECT * FROM pg_getrlimit('v');
SELECT length(repeat(string_agg(md5(g::text), ','),1024)) FROM generate_series(1, 10000) g;
\echo You can ignore the diff of detail messages
