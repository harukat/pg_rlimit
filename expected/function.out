SELECT * FROM pg_setrlimit('v', -1);
 pg_setrlimit 
--------------
           -1
(1 row)

SELECT * FROM pg_getrlimit('v');
 pg_getrlimit 
--------------
           -1
(1 row)

SELECT * FROM pg_setrlimit('v', 512 * 1024 * 1024);
 pg_setrlimit 
--------------
    536870912
(1 row)

SELECT * FROM pg_getrlimit('v');
 pg_getrlimit 
--------------
    536870912
(1 row)

SELECT length(repeat(string_agg(md5(g::text), ','),1024)) FROM generate_series(1, 10000) g;
ERROR:  out of memory
DETAIL:  Failed on request of size 337918980 in memory context "ExprContext".
\echo You can ignore the diff of detail messages
You can ignore the diff of detail messages
